### ISSUE TITLE
WWA building order.

### ISSUE THREAD

**Issue by papaathome** [abrudana/wwa#5]
Dear Sir,

I found a potential problem in WWA.sln, the building order is not specified.
If you select from the menu [Project][Project dependencies] the 'Project Dependencies' form is opened.
In the drop-down list 'Projects' in tab [Depenencies] you can select each project in the solution and specify on which other projects it depends, specifying a build order. (Which you can inspect in the other tab.)

I found in my version of the WWA sources that no dependency was specified.
The way I see it, technically this is a problem. Project 'WWA' could be build before project 'SOFA' is ready and in the best case the linker in 'WWA' will complain about a missing dll to link against but in the worse case it would link against an old and stale dll. (Which will make bug-hunting a real difficult process!)

I specified the following dependencies:
SOFA: none
WWA: [v] SOFA
WWATest: [v] WWA
That fix it for me.

It can be that the source I used (WWA-8.zip) is somehow missing this data.
Can you inspect your original sources and correct if necessary?

Kind regards, André,

**Comment by abrudana** [comment]
Dear André,
My version contains [Build Order]. It was always good on my side, cause I usually do the following: 

1. Clean project
2. Rebuild solution

I never experienced such a problem as you.

Anyway your source code are not the latest. Please use the latest.
Regards,
Attila

**Comment by papaathome** [comment]
Dear Attila,

I also had no problem with it, because I never changed any actual code.

I usually do only a 'rebuild' at solution level and I had a failing build where the WWA project could not find the SOFA output. That triggered me to check the build order.

I have cloned a new git copy, no longer using the .zip version. I will keep the .zip version in my archives as backup for what I did for the NuGet package.
I'm not merging in any results from the .zip version, waiting for your actions and then get an update for it.

Kind regards,
André.
